spec/nexpose/connection/site_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Nexpose::Connection, :vcr do
+  let(:console_hostname) { ENV['NEXPOSE_HOSTNAME'] }
+  let(:username) { ENV['NEXPOSE_USERNAME'] }
+  let(:password) { ENV['NEXPOSE_PASSWORD'] }
+  let(:connection) do
+    Nexpose::Connection.new(console_hostname, username, password).tap(&:login)
+  end
+
+  describe 'Site methods' do
+    describe '#sites' do
+      attr_reader :expected_sites
+
+      after do
+        VCR.use_cassette('delete_basic_site') do
+          basic_site.delete(connection)
+        end
+      end
+
+      before do
+        @expected_sites = [basic_site]
+      end
+
+      let(:basic_site) do
+        Nexpose::Site.new("test site name 1").tap do |site|
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.